ledger-bridge.js

```diff
@@ -140,11 +140,11 @@ export default class LedgerBridge {
         }
     }
 
-    async signTransaction (replyAction, hdPath, tx, to) {
+    async signTransaction (replyAction, hdPath, tx, to, chainId) {
         try {
             await this.makeApp()
-            if (to) {
-                const isKnownERC20Token = byContractAddress(to)
```

Comment:
I think we still need to call this if `chainId` is not provided.

Reply:
`byContractAddress` no longer exists; do we want to have a default `chainId`?

Reply:
It does appear to still exist: https://github.com/LedgerHQ/ledgerjs/blob/v6.4.1/packages/hw-app-eth/src/erc20.ts#L6

Reply:
@Gudahtt Why did you think that we still need to call this if `chainId` is not provided?

Reply:
When I made this comment, it had been replaced with a call to `byContractAddressAndChainId` if `chainId` was provided, but the block was skipped entirely if `chainId` was omitted. This was how the contract information was passed to the Ledger device, so it could show a more informative prompt to the user. It appeared at the time that it would result in the contract metadata not being sent on older MetaMask versions (i.e. versions that didn't send the `chainId`).

Since then, we discovered that `byContractAddressAndChainId` replaced `byContractAddress` in v6.5.0. It was an undocumented breaking change. So @darkwing updated the range to ensure v6.5.0 wasn't used, so that we could keep using `byContractAddress` and maintain the same behavior we have in production. Then we could update to v6.5.0 once the version of MetaMask that sent a `chainId` with the message had been in the wild long enough that we were comfortable breaking this functionality for users of outdated versions.

Also this whole operation might be unnecessary now, as I mentioned here: https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/95#discussion_r690552786
Not totally sure I understand that change though.